lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -330,6 +329,10 @@ def add_3d_secure(post, options)
 
       def add_3d_secure_1_data(post, options)
         post[:i8] = build_i8(options[:eci], options[:cavv], options[:xid])
+        if options[:three_ds_version]
+          three_ds_version = options[:three_ds_version] == '1' ? '1.0' : options[:three_ds_version]
+          post[:'3ds_version'] = three_ds_version
+        end
```

Comment:
This behavior was a bit unexpected - based on what we heard from the gateway, it seemed like any request where i8 was passed would need to pass version as well. However, 3ds1 transactions seem to work with or without it. Added this logic to include it when present, but open to other thoughts.

Reply:
Based on what the gateway says, do you think we should send a default value if `i8` is included and no `options[:three_ds_version]` is specified?

Reply:
yeah, i think you're right - even though it seems to work in the test env the new docs explicitly say it's mandatory with i8. will update, thank you!